src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/DownloadModal/components/DownloadButton/index.tsx

```diff
@@ -55,31 +50,10 @@ const DownloadButton = ({
 
   const [tsvData, setTsvData] = useState<string[][]>([]);
 
-  const tableRefactorFlag = useTreatments([USER_FEATURE_FLAGS.table_refactor]);
-  const usesTableRefactor = isUserFlagOn(
-    tableRefactorFlag,
-    USER_FEATURE_FLAGS.table_refactor
-  );
-
   useEffect(() => {
-    if (usesTableRefactor) {
-      const newTsvData = mapTsvData(checkedSamples);
-      setTsvData(newTsvData);
```

Comment:
does anyone remember why this was behind the feature flag? It seems to me that this function is working as expected now but i wanted to make sure i wasn't missing something in the larger picture.

Reply:
Maya duplicated (and I think deduplicated) a bunch of components from various modals.  Could this be one of them?  

If this change works with and without the feature flag on, I guess it's ok.  I'm not in a hurry to remove this since the table refactor is still in progress.